### PR TITLE
371 pjg update setup demos

### PR DESCRIPTION
**Pull request by gleckler1**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -51,13 +51,13 @@
     "data/CMIP_metrics_results/CMIP5/historical/*.json")
```
Comment (durack1) [inline setup.py]: @gleckler1 @doutriaux1 I've just had a quick look at this and it needs a fairly significant overhaul considering we've now moved to a conda install path.. None of the subdirs make much sense - as in if these are not pulled across in the conda installation there's no need to include them in the setup.py - I'll create a new issue for this

**Comment by durack1** [comment]
@doutriaux1 with the `master` and edits to `.travis.yml` in this branch it's now got conflicts - if I keep going it's just going to get even more tangled so handing it over to you..

**Comment by doutriaux1** [comment]
no problem.

**Comment by gleckler1** [comment]
@doutriaux1 PG to prepare demos.   Once that is done CD to ensure they map properly into conda

**Comment by gleckler1** [comment]
@doutriaux1 @durack1  In order for PG to follow through with demo we need to get this PR corrected/merged so the rest of the demos can follow from it...

**Comment by gleckler1** [comment]
@doutriaux1 closing this PR because it is partially redundant with https://github.com/PCMDI/pcmdi_metrics/pull/376

**Pull request closed without merging**